spec/StringClassSpec.js

```diff
@@ -0,0 +1,172 @@
+  });
+});
+
+describe('test isQuestion method', () => {
+  it('should return true if the string is a question', () => {
+    expect('TONIDA HAS ORANGES?'.isQuestion()).toBeTruthy();
+    expect('TonidA?'.isQuestion()).toBeTruthy();
+    expect('M37@i*, exp?'.isQuestion()).toBeTruthy();
+    expect('mabishi?'.isQuestion()).toBeTruthy();
+  });
+
+  it('should return false if the string is not a question', () => {
+    expect('TBNRTY'.isQuestion()).toBeFalsy();
+    expect('tbnrty'.isQuestion()).toBeFalsy();
+    expect('245Gy'.isQuestion()).toBeFalsy();
+    expect('Thy'.isQuestion()).toBeFalsy();
+  });
+
+  it('should check the type of results to be boolean', () => {
+    expect(typeof('mabishi?'.isQuestion())).toBe('boolean');
+    expect(typeof('TBNRTY'.isQuestion())).toBe('boolean');
+  });
+});
+
+describe('test words function', () => {
+  it('should return an array of the string provided', () => {
+    expect('tonida'.words()).toEqual(['tonida']);
+    expect('Angel:Gabriel,is going home'.words())
+    .toEqual(['Angel', 'Gabriel', 'is', 'going', 'home']);
+    expect('Today,tomorrow'.words()).toEqual(['Today', 'tomorrow']);
+    expect(''.words()).toEqual(null);
+    expect('This is Andela'.words()).toEqual(['This', 'is', 'Andela']);
+  });
+
+  it('should check that the type of results is an object', () => {
+    expect(typeof('tonida'.words())).toEqual('object');
+  });
+
+  it('should check the results to be an instance of an array', () => {
+    expect(Array.isArray('Andela'.words())).toBeTruthy();
+  });
+});
+
+describe('test wordCount method', () => {
+  it('should count the number of words in a string', () => {
+    expect('America'.wordCount()).toEqual(1);
+    expect('This really sucks'.wordCount()).toEqual(3);
+    expect('I:love, GOD'.wordCount()).toEqual(3);
+  });
+
+  it('should check type of the results to be number', () => {
+    expect(typeof('America'.wordCount())).toBe('number');
+  });
+});
+
+describe('test toCurrency method', () => {
+  it('should return a currency representation of the string', () => {
+    expect('12364'.toCurrency()).toEqual('12,364.00');
+    expect('450425.97'.toCurrency()).toEqual('450,425.97');
+    expect('8670'.toCurrency()).toEqual('8,670.00');
+  });
+
+  it('should check the type of results to be a string', () => {
+    expect(typeof('12364'.toCurrency())).toBe('string');
+  });
+});
+
+describe('test fromCurrency method', () => {
+  it('should return a number format of the string', () => {
+    expect('11,111.11'.fromCurrency()).toEqual(11111.11);
+    expect('1,111.00'.fromCurrency()).toEqual(1111.00);
+    expect('1,111'.fromCurrency()).toEqual(1111);
+  });
+
+  it('should check the type of results to be number', () => {
+    expect(typeof('11,111.11'.fromCurrency())).toBe('number');
+  });
+});
+
+describe('test numberWords method', () => {
+  it('should return the numbers in words', () => {
+    expect('345'.numberWords()).toEqual('three four five ');
```

Comment:
**Nitpick**
- Could you find a way to remove the extra space at the end of the words?